tensorflow/lite/CMakeLists.txt

```diff
@@ -298,6 +300,9 @@ if(_TFLITE_ENABLE_NNAPI)
   populate_tflite_source_vars(
     "nnapi" TFLITE_NNAPI_SRCS FILTER "(_disabled)\\.(cc|h)$"
   )
+  if(${TFLITE_ENABLE_NNAPI_VERBOSE_VALIDATION})
+    list(APPEND TFLITE_TARGET_PUBLIC_OPTIONS "-DNNAPI_VERBOSE_VALIDATION")
```

Comment:
NNAPI_VERBOSE_VALIDATION doesn't look used in the nnapi_delegate.cc.
Also could you also update Bazel build file?

Reply:
The NNAPI_VERBOSE_VALIDATION ifdef's the implementation of the AddValidationFailure() and AddValidationFailureFmt():
https://github.com/tensorflow/tensorflow/blob/458234a3d34326d353814be0d4f85f6713c59ea0/tensorflow/lite/delegates/nnapi/nnapi_delegate.cc#L1529-L1539
https://github.com/tensorflow/tensorflow/blob/458234a3d34326d353814be0d4f85f6713c59ea0/tensorflow/lite/delegates/nnapi/nnapi_delegate.cc#L1542-L1555

The bazel is up-to-date already. It contains an aditional cc library (nnapi_delegate_verbose_validation), including this option:
https://github.com/tensorflow/tensorflow/blob/458234a3d34326d353814be0d4f85f6713c59ea0/tensorflow/lite/delegates/nnapi/BUILD#L46-L65